editor/resources/templates/template.collection

```diff
@@ -1 +1 @@
-name: "default"
+name: "__NAME__"
```

Comment:
We use __KEY__ elsewhere. Or should we use %KEY% or {{KEY}} instead?

Reply:
I would prefer us to use `{{name}}` or `{{NAME}}` tbh, I think this is a widely used templating syntax.

Reply:
I've changed it to `{{NAME}}`!